go/mysql/proto/structs.go

```diff
@@ -41,10 +41,39 @@ const (
 	VT_GEOMETRY    = 255
 )
 
+// MySQL field flags bitset values e.g. to distinguish between signed and unsigned integer.
+// Comments are taken from the original source code.
+// These numbers should exactly match values defined in dist/mysql-5.1.52/include/mysql_com.h
+const (
+	// VT_ZEROVALUE_FLAG is not part of the MySQL specification and only used in unit tests.
+	VT_ZEROVALUE_FLAG    = 0
+	VT_NOT_NULL_FLAG     = 1   /* Field can't be NULL */
```

Comment:
On second thoughts. It's better if these are explicit values if you copied them from the .h file.
Ideally, you should do what I did here: https://github.com/youtube/vitess/blob/master/go/mysql/mysql.go#L41.
But that will introduce a C dependency in this package. So, I think things are fine as is for now.